presto-main/src/main/java/com/facebook/presto/operator/DeleteOperator.java

```diff
@@ -29,17 +30,19 @@
 import java.util.Optional;
 import java.util.function.Supplier;
 
+import static com.facebook.presto.operator.OperatorExecutionContext.OPERATOR_EXECUTION_CONTEXT_JSON_CODEC;
 import static com.facebook.presto.spi.type.BigintType.BIGINT;
 import static com.facebook.presto.spi.type.VarbinaryType.VARBINARY;
 import static com.google.common.base.Preconditions.checkState;
 import static io.airlift.concurrent.MoreFutures.getFutureValue;
 import static io.airlift.concurrent.MoreFutures.toListenableFuture;
+import static io.airlift.slice.Slices.wrappedBuffer;
 import static java.util.Objects.requireNonNull;
 
 public class DeleteOperator
         implements Operator
 {
-    public static final List<Type> TYPES = ImmutableList.of(BIGINT, VARBINARY);
+    public static final List<Type> TYPES = ImmutableList.of(BIGINT, VARBINARY, VARBINARY);
```

Comment:
I think now is a good time for us to have comments about what each channel is for now :)